test/graph_rewrites/test_ng_var_update_ng_tensor.cc

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include <vector>
+
+#include "gtest/gtest.h"
+#include "tensorflow/core/framework/allocator.h"
+#include "tensorflow/core/framework/node_def_builder.h"
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/resource_mgr.h"
+#include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/framework/tensor_testutil.h"
+#include "tensorflow/core/framework/types.h"
+#include "tensorflow/core/framework/types.pb.h"
+#include "tensorflow/core/kernels/ops_testutil.h"
+#include "tensorflow/core/kernels/ops_util.h"
+#include "tensorflow/core/lib/core/status_test_util.h"
+#include "tensorflow/core/platform/test.h"
+
+#include "ngraph_bridge/enable_variable_ops/ngraph_var.h"
+#include "ngraph_bridge/enable_variable_ops/ngraph_variable_update_ng_tensor_op.h"
+#include "ngraph_bridge/enable_variable_ops/tf_fake_input.h"
+#include "ngraph_bridge/ngraph_utils.h"
+#include "test/test_utilities.h"
+
+namespace tensorflow {
+namespace ngraph_bridge {
+namespace testing {
+
+class NGVarUpdateNGTensorOpTest : public tensorflow::OpsTestBase {};
+
+// This test requires the env variable NGRAPH_TF_NGVARIABLE_BUFFER_SHARING=0
+// when running on CPU
+TEST_F(NGVarUpdateNGTensorOpTest, KernelTest) {
```

Comment:
Is it possible to replace `NGVarUpdateNGTensorOpTest` with `tensorflow::OpsTestBase` here? and then remove line 43?

Reply:
stale comment